reaper/cogs/log_reading.py

```diff
@@ -201,7 +202,7 @@ async def on_message(self, message):
                     callback = True
 
                 # Check if the OLD, merged better server browser is loading. It's broken now and causes issues
-                if "Better.Serverbrowser" in line:
+                elif "Better.Serverbrowser" in line:
```

Comment:
Wouldn't multiple `if` statements make more sense as opposed to `elif` as we can find multiple issues in the log? Or am I missing something here? ^^"

Reply:
when checking each line of the log only one of these if statements can be true
for example if the first if statement passes, it's not necessary to check the rest of them, since this line is essentially done being checked

Reply:
Ah, okay, nevermind I see. Yeah tbh this should just be a regex over the whole file rather than a line-by-line thing lol. But that's a separate PR anyway ^^